src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicNativeMethods.cs

```diff
@@ -609,35 +598,8 @@ internal struct SOCKADDR_IN6
             internal ushort _family;
             internal ushort _port;
             internal uint _flowinfo;
-            internal byte _addr0;
-            internal byte _addr1;
-            internal byte _addr2;
-            internal byte _addr3;
-            internal byte _addr4;
-            internal byte _addr5;
-            internal byte _addr6;
-            internal byte _addr7;
-            internal byte _addr8;
-            internal byte _addr9;
-            internal byte _addr10;
-            internal byte _addr11;
-            internal byte _addr12;
-            internal byte _addr13;
-            internal byte _addr14;
-            internal byte _addr15;
+            internal fixed byte _addr[16];
```

Comment:
As long as you're changing the field, can you rename it to sin6_addr?  Ideally the rest of the fields would be renamed to match the native definition as well, but that's not necessary for your PR.